### PR TITLE
install_amp.sh: use DEB

### PR DESCRIPTION
**Pull request by aledsage**

### CODE REVIEW

**Comment by m4rkmckenna** [comment]
LGTM Tested 

I have added `sudo iptables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to-ports 8081` to my local vagrant setup saves having to type `:8081`

File: files/install_amp.sh [hunk 1]
```diff
@@ -39,55 +38,28 @@ echo "Restarting Syslog"
...
-echo "Installing Karaf wrapper service"
-cd /home/vagrant/cloudsoft-amp-karaf
-echo ".. starting karaf manually"
-./bin/start
-echo ".. wait for karaf to start"
-while ! ./bin/client version > /dev/null 2>&1; do
-  echo ".... waiting for 5 seconds"
-  sleep 5
+echo "Waiting for AMP to start..."
+while ! (sudo grep "BundleEvent STARTED - org.apache.brooklyn.karaf-init" /opt/amp/log/amp.debug.log) > /dev/null ; do
```
Comment (murdoaird) [inline files/install_amp.sh]: @aledsage This line fails (but overall install is ok) as the file is not created at this point (I think)

`==> amp: grep: /opt/amp/log/amp.debug.log
==> amp: : No such file or directory`

Also, FYI, users access the vm as the 'vagrant' user but that user cannot access the log files (as restricted to 'amp'). Assuming I'll just update docs to prefix with 'sudo'?

Comment (aledsage) [inline files/install_amp.sh]: Yes, please update docs.

Does it keep retrying though? It worked for me (I got some stderr to say that the file didn't exist, and then it went round the loop again to try again 2 seconds later).

**Comment by aledsage** [comment]
@murdoaird has confirmed that this does retry, so merging PR now.

**Pull request merged**